stdlib/array.gr

```diff
@@ -242,7 +254,14 @@ provide let copy = array => {
  * @param fn: The iterator function to call with each element
  * @param n: The number of times to iterate the given array
  * @param array: The array to iterate
- *
+ * 
+ * @example
+* @example
```

Comment:
It looks like the indenting here might have gotten messed up. It could just be github beind weird though too. IT looks like the `*`s are in a space which is different then the rest and also there is two spaces between some of the `*`s and the code which is different then the rest of the examples